### PR TITLE
[Tests-Only][CI] fix the starlark code for reduced ci after merge and sonarcloud

### PR DESCRIPTION
**Pull request by dpakach**
fix the starlark code for reduced ci after merge and sonarcloud

issue https://github.com/owncloud/QA/issues/649

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,9 @@ vendor-bin/**/composer.lock
 /tests/acceptance/output*
 /tests/output
 .phpunit.result.cache
+
+# SonarCloud scanner
+.scannerwork
+
+# SonarCloud scanner
+.scannerwork
```
Comment (phil-davis) [inline .gitignore:22]: duplicated lines

**Comment by mmattel** [comment]
@phil-davis may I ping :smile:

**Comment by phil-davis** [comment]
> @phil-davis may I ping smile

@mmattel you may. I have about 30 PRs in the various oC10 app repos. Hopefully I get to look at them all by the end of tomorrow!

**Comment by mmattel** [comment]
> I have about 30 PRs in the various oC10 app repos. Hopefully I get to look at them all by the end of tomorrow!

Superhero of the week ! :superhero_man:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**